### PR TITLE
add js for quality selector explanation

### PR DESCRIPTION
**Pull request by OmgImAlexis**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

@fernandog

### CODE REVIEW

File: views/inc_qualityChooser.mako [hunk 1]
```diff
@@ -40,14 +40,14 @@ selected = None
...
+            <h5>Downloads <b>any</b> of these qualities:
                 ${', '.join([Quality.qualityStrings[i] for i in allowed_qualities + preferred_qualities])}.</h5>
-            <h5>But will <b>stop searching</b> when find <b>any</b> from: ${', '.join([Quality.qualityStrings[i] for i in preferred_qualities])}</h5>
-                <b>Note:</b> Status from Preferred quality will be SNATCHED BEST, else SNATCHED.
+            <h5>Stop search only when downloads <b>any</b> from Preferred: ${', '.join([Quality.qualityStrings[i] for i in preferred_qualities])}</h5>
         % else:
-            <h5>Downloads <b>any</b> of this qualities: ${', '.join([Quality.qualityStrings[i] for i in allowed_qualities])} and stop searching.</h5>
-            <b>Note:</b> Status will be SNATCHED.
+            <h5>Downloads <b>any</b> of these qualities: ${', '.join([Quality.qualityStrings[i] for i in allowed_qualities])}</h5>
+            <h5>Stop search</h5>
```
Comment (OmgImAlexis) [inline views/inc_qualityChooser.mako]: This should be fixed as "stop search" doesn't really make much sense on it's own.

File: static/js/home/edit-show.js [hunk 2]
```diff
@@ -16,13 +16,15 @@ MEDUSA.home.editShow = function() {
...
-                html += '<h5>But will <b>stop searching</b> when find <b>any</b> from: ' + preferred + '</h5>';
-                html += '<b>Note:</b> Status from Preferred quality will be SNATCHED BEST, else SNATCHED.';
+                html += '<h5>Downloads <b>any</b> of these qualities: ' + both + '</h5>';
+                html += '<h5>Stop search only when downloads <b>any</b> from Preferred: ' + preferred + '</h5>';
             $('#quality_explanation').html(html);
         } else {
-            $('#quality_explanation').html('<h5>Downloads <b>any</b> of these qualities: ' + qualities + ' and stop searching.</h5><b>Note:</b> Status will be SNATCHED.');
+                html += '<h5>Downloads <b>any</b> of these qualities: ' + allowed + '</h5>';
+                html += '<h5>Stop search</h5>'
+            $('#quality_explanation').html(html);
```
Comment (OmgImAlexis) [inline static/js/home/edit-show.js]: "Stop search" same as below.

File: static/js/home/edit-show.js [hunk 3]
```diff
@@ -16,13 +16,15 @@ MEDUSA.home.editShow = function() {
         var both = allowed.concat(preferred.filter(function (item) {
             return allowed.indexOf(item) < 0;
         }));
+        var html = '<h5><b>Quality setting explanation:</b></h5>'
         if (preferred.length) {
-            var html = '<h5>Downloads <b>any</b> of this qualities: ' + both + '</h5>';
-                html += '<h5>But will <b>stop searching</b> when find <b>any</b> from: ' + preferred + '</h5>';
-                html += '<b>Note:</b> Status from Preferred quality will be SNATCHED BEST, else SNATCHED.';
+                html += '<h5>Downloads <b>any</b> of these qualities: ' + both + '</h5>';
+                html += '<h5>Stop search only when downloads <b>any</b> from Preferred: ' + preferred + '</h5>';
```
Comment (OmgImAlexis) [inline static/js/home/edit-show.js]: Same again.

**Comment by fernandog** [comment]
1itmv1

**Comment by fernandog** [comment]
@OmgImAlexis are you going to work on this today?  thanks!

**Comment by OmgImAlexis** [comment]
@fernandog if you're happy with this then merge.

**Comment by fernandog** [comment]
What do you think? Is it clear? You said you have suggestions.

Also this will conflic with UI Changes PR as @thraxis moved away code from this JS we changed. So better merge after that right?

**Comment by OmgImAlexis** [comment]
Yeah check the last commit, I just changed it. It should be fine to merge BEFORE his.

**Comment by fernandog** [comment]
@medariox ok for 0.1.10? You are the one to say no more features =)

**Pull request merged**